expressionGenerator.js

```diff
@@ -0,0 +1,51 @@
+const fieldsToUpdate = (body) => {
+    var fields = ['name', 'description', 'showDate']
+    var fieldsToUpdate = []
+    var expressions = []
+    const bodyFields = Object.keys(body)
+
+    for (let index = 0; index < bodyFields.length; index++) {
+        if (fields.includes(bodyFields[index])) {
+            fieldsToUpdate.push(bodyFields[index])
+        }
+    }
+
+    const generateUpdateExpression = () => {
+        var updateExpressionString = 'set'
+
+        for (let index = 0; index < fieldsToUpdate.length; index++) {
+            if (index == fieldsToUpdate.length - 1) {
+                updateExpressionString = updateExpressionString.concat(' #old' + fieldsToUpdate[index].toUpperCase() + ' = :new' + fieldsToUpdate[index].toUpperCase())
+            } else {
+                updateExpressionString = updateExpressionString.concat(' #old' + fieldsToUpdate[index].toUpperCase() + ' = :new' + fieldsToUpdate[index].toUpperCase() + ',')
+            }
+        }
```

Comment:
Não faz sentido usar esse if.
Outro ponto também da uma conferida como funciona o map do JS, vou adicionar um exemplo pra esse trecho usando o próprio for pra ajudar aqui, mas faz sentido usar o map em muitos cenários.

```suggestion
        
        for (let index = 0; index < fieldsToUpdate.length; index++) {
                updateExpressionString = updateExpressionString.concat(' #old' + fieldsToUpdate[index].toUpperCase() + ' = :new' + fieldsToUpdate[index].toUpperCase() + ',')
        }
        updateExpressionString = updateExpressionString.substring(0, updateExpressionString.length - 1);
```